src/components/post-comment.jsx

```diff
@@ -79,11 +79,11 @@ export default class PostComment extends React.Component {
 
     return (
     <div className="comment">
-      <a className="comment-icon fa fa-comment-o"
+      {this.props.omitBubble ? false : <a className="comment-icon fa fa-comment-o"
```

Comment:
On code style: could you please make it consistent with other similar structures:

```
{!this.props.omitBubble ? (
  <a className="comment-icon fa fa-comment-o"
  ...
) : false}